.circleci/config.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           command: |
             ./configure.sh
             ./gradlew --no-daemon dependencies
-            ./gradlew --no-daemon --stacktrace build -x test
+            ./gradlew --no-daemon --stacktrace build -x test -x integrationTest
```

Comment:
shouldn't we exclude int. tests in this [line](https://github.com/rsksmart/rskj/blob/f35bf63c34fac76acbbe59cd27773572ae5df204/.circleci/config.yml#L63) too?

Reply:
Done